index.html

```diff
@@ -50,28 +50,61 @@
         will be added to this document as they are implemented across multiple user agents.
       </p>
       <p>
-        Please subscribe to the <a href="https://github.com/w3c/permissions-registry/">Powerful
-        Features Registry Repository</a> on GitHub to be notified of changes.
+        Please subscribe to the <a href="https://github.com/w3c/permissions-registry/">Permissions
+        Registry Repository</a> on GitHub to be notified of changes.
+      </p>
+    </section>
+    <section>
+      <h2>
+        Purpose
+      </h2>
+      <p>
+        The purpose of this is to provide a centralized place where to find the [=policy-controlled
+        features=] and/or [=powerful features=] of the web platform.
+      </p>
+      <p>
+        By splitting the registry into standardized permissions and provisional permissions, the
+        registry also provides a way to track the status of these features.
       </p>
     </section>
     <section>
       <h2>
         Change Process
       </h2>
       <p>
-        The process for adding and/or updating a [=powerful feature=] in this registry is as
-        follows:
+        The process for adding and/or updating this registry is as follows:
       </p>
       <ol>
-        <li>Determine if your feature meets the definition of a [=powerful feature=] (i.e.,
-        requires [=express permission=] to be used).
+        <li>If necessary, add a "Permissions Policy" section to your specification which includes
+        the following:
+          <ol>
+            <li>The token string that identifies the policy controlled feature (e.g.,
+            `"super-awesome"`). Make sure the token string is linkable by wrapping it a [^dfn^]
```

Comment:
should we mention exporting the dfn?

Reply:
I didn't want to mention "export" because it puts a dependency on the (ReSpec/Bikeshed) tooling. We should definitely mention it in the pull request template tho.